spylib/oauth/validations.py

```diff
@@ -18,25 +13,8 @@ def validate_callback(shop: str, timestamp: int, query_string: Any, api_secret_k
         raise ValueError('Timestamp is too old')
 
     # 3) Check the hmac
-    # Extract HMAC
-    args = parse_qsl(q_str)
-    original_args = deepcopy(args)
-    try:
-        # Let's assume alphabetical sorting to avoid issues with scrambled args when using bonnette
-        args.sort(key=itemgetter(0))
-        validate_callback_args(args=args, api_secret_key=api_secret_key)
-    except ValueError:
-        # Try with the original ordering
-        validate_callback_args(args=original_args, api_secret_key=api_secret_key)
-
-
-def validate_callback_args(args: List[Tuple[str, str]], api_secret_key: str) -> None:
-    # We assume here that the arguments were validated prior to calling
-    # this function.
-    hmac_arg = [arg[1] for arg in args if arg[0] == 'hmac'][0]
-    message = '&'.join([f'{arg[0]}={arg[1]}' for arg in args if arg[0] != 'hmac'])
-    # Check HMAC
-    validate_hmac(secret=api_secret_key, sent_hmac=hmac_arg, message=message)
```

Comment:
This functionality in this function was moved to `hmac.py` as the `validate_hmac` function wasn't doing anything before.